lib/actions-oclif/deploy.ts

```diff
@@ -243,30 +243,30 @@ ${dockerignoreHelp}
 				compositionToBuild.services,
 				servicesToSkip,
 			);
+			let builtImagesByService: Dictionary<any> = {};
 			if (_.size(compositionToBuild.services) === 0) {
 				logger.logInfo(
 					'Everything is up to date (use --build to force a rebuild)',
 				);
-				return {};
```

Comment:
This `return` statement made sense when this block was in a promise chain, but should have been removed in the conversion to async/away. Check this line in PR #2013:
https://github.com/balena-io/balena-cli/pull/2013/files#diff-cd710231a0e1573249414ab3ebf325c7L253